spec/grape/middleware/formatter_spec.rb

```diff
@@ -213,7 +213,13 @@ def to_xml
   context 'no content responses' do
     let(:no_content_response) { ->(status) { [status, {}, ['']] } }
 
-    Rack::Utils::STATUS_WITH_NO_ENTITY_BODY.each do |status|
+    STATUSES_WITHOUT_BODY = if Rack::RELEASE >= '2.1.0'
```

Comment:
Use `Gem::Version.new` to compare, otherwise we're going to be in trouble with version 11.

https://medium.com/little-programming-joys/how-to-compare-version-strings-in-ruby-ced0916a4b71

Reply:
Sure, I'll fix it. 
You did in this way here: https://github.com/ruby-grape/grape/blob/master/spec/grape/integration/rack_spec.rb#L23 So I've assumed it's how you would want it here too.

Reply:
That link doesn't go anywhere interesting ;) If you see it done elsewhere, please also fix!

Reply:
It did until you've merged my other PR :)
But I'll fix this today, thank you